components/admin/requests/forms/new-applications/PhysicianAssessmentForm.tsx

```diff
@@ -0,0 +1,188 @@
+ *
+ * @param {PhysicianAssessmentInformation} physicianAssessmentInformation Data Structure that holds all physician assessment information for a client request.
+ * @param onChange Function that uses the updated values from form.
+ */
+export default function PhysicianAssessmentForm({
+  physicianAssessmentInformation,
+  onChange,
+}: PhysicianAssessmentFormProps) {
+  return (
+    <>
+      {/* Personal Information Section */}
+      <Box paddingBottom="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {`Physician’s Assessment`}
+        </Text>
+        <Stack spacing="20px">
+          <FormControl isRequired>
+            <FormLabel>{'Medical name of disabling condition(s)'}</FormLabel>
+            <Input
+              value={physicianAssessmentInformation?.disability}
+              onChange={event =>
+                onChange({
+                  ...physicianAssessmentInformation,
+                  disability: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl isRequired>
+            <FormLabel>{'Physician’s certification date'}</FormLabel>
+            <Input
+              type="date"
+              value={physicianAssessmentInformation.physicianCertificationDate}
+              onChange={event =>
+                onChange({
+                  ...physicianAssessmentInformation,
+                  physicianCertificationDate: event.target.value,
+                })
+              }
+            />
+            <FormHelperText color="text.seconday">{'Format: YYYY-MM-DD'}</FormHelperText>
+          </FormControl>
+        </Stack>
+      </Box>
+
+      <Divider borderColor="border.secondary" />
+
+      {/* Patient Eligibility Section */}
+
+      <Box paddingY="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {'Patient Eligibility'}
+        </Text>
+
+        <Stack spacing="20px">
+          <FormControl isRequired>
+            <FormLabel>{'Please select the condition'}</FormLabel>
+            <RadioGroup
+              value={
+                physicianAssessmentInformation.affectsMobility
+                  ? '0'
+                  : physicianAssessmentInformation.cannotWalk100m
+                  ? '1'
+                  : physicianAssessmentInformation.mobilityAidRequired
+                  ? '2'
+                  : '3'
+              }
+              onChange={value => {
+                // handleChangedPatientEligibility
+                onChange({
+                  ...physicianAssessmentInformation,
+                  affectsMobility: value === '0',
+                  cannotWalk100m: value === '1',
+                  mobilityAidRequired: value === '2',
+                  ...(value !== '3' && {
+                    patientEligibilityDescription: undefined,
+                  }),
+                });
+              }}
+            >
```

Comment:
Getting design clarification regarding whether this may be a checkbox field instead. Will update this thread

Reply:
Okay this will remain as a checkbox field. Can you add a TODO comment here to revise the logic/DB schema? We need to replace our 3 boolean columns in our DB with a single enum column. That will be a breaking change.

For now, let's merge this code in